modules/custom-status/custom-status.php

```diff
@@ -416,32 +416,14 @@ function post_admin_header() {
 
 			$custom_statuses = apply_filters( 'ef_custom_status_list', $custom_statuses, $post );
 
-			// All right, we want to set up the JS var which contains all custom statuses
-			$all_statuses = array();
-
-			// The default statuses from WordPress
-			$all_statuses[] = array(
-				'name' => __( 'Published', 'edit-flow' ),
-				'slug' => 'publish',
-				'description' => '',
-			);
-			$all_statuses[] = array(
-				'name' => __( 'Privately Published', 'edit-flow' ),
-				'slug' => 'private',
-				'description' => '',
-			);
-			$all_statuses[] = array(
-				'name' => __( 'Scheduled', 'edit-flow' ),
-				'slug' => 'future',
-				'description' => '',
-			);
-
-			// Load the custom statuses
-			foreach( $custom_statuses as $status ) {
-				$all_statuses[] = array(
-					'name' => esc_js( $status->name ),
-					'slug' => esc_js( $status->slug ),
-					'description' => esc_js( $status->description ),
+			$all_statuses = array_merge( $custom_statuses, $this->get_default_statuses() );
```

Comment:
+1 to abstracting this into `get_default_statuses()`

Reply:
Still kinda confusing, but hoping to move in the right direction